tensorflow/compiler/xla/service/gpu/launch_dimensions.h

```diff
@@ -29,24 +29,37 @@ namespace gpu {
 // number of threads per block.
 class LaunchDimensions {
```

Comment:
Could you provide some more info in the PR description why 3-dimensions are needed for the launch dimensions?

Reply:
I added more info in the PR description and more corresponding comments in the codes. Let me know if it helps.